en_US/access-control/authz/authz.md

```diff
@@ -22,7 +22,13 @@ The process of the authorization check is as follows:
    - If yes, EMQX switches to the next authenticator to continue the process.
    - If it is already the last authorizer, EMQX follows the setting of `no_match` to determine whether to allow or reject the client operation.
 
-For information on how to adjust the sequence of the authorizer in an authorization chain and how to check the running metrics, see [Manage authorizers](#manage-authorizers).
+::: tip Warning
+
+To avoid problems with the authorization, you need to remember to remove the authorizer using ACL file when necessary because it has `{allow, all}` at the end by default. 
```

Comment:
```suggestion
```suggestion
To avoid problems with the authorization, you need to remember to remove the  ACL file authorizer when necessary because it has `{allow, all}` at the end by default. 
```